Fix "java.lang.Exception: Manifest error: Missing attribute name -  sun.security.action,sun.security.internal.spec,sun.security.jgss" during boot

When starting up a GF build from the current master, an exception is logged:

```
WARNING: Failed to install file:/glassfish6/glassfish/modules/glassfish-extra-jre-packages.jar.
org.osgi.framework.BundleException: Could not create bundle object.
	at org.apache.felix.framework.Felix.installBundle(Felix.java:3312)
	at org.apache.felix.framework.BundleContextImpl.installBundle(BundleContextImpl.java:147)
	at com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.install(BundleProvisioner.java:420)
	at com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.installBundles(BundleProvisioner.java:180)
	at com.sun.enterprise.glassfish.bootstrap.osgi.OSGiGlassFishRuntimeBuilder.build(OSGiGlassFishRuntimeBuilder.java:118)
	at org.glassfish.embeddable.GlassFishRuntime._bootstrap(GlassFishRuntime.java:133)
	at org.glassfish.embeddable.GlassFishRuntime.bootstrap(GlassFishRuntime.java:86)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishMain$Launcher.launch(GlassFishMain.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishMain.main(GlassFishMain.java:73)
	at com.sun.enterprise.glassfish.bootstrap.ASMain.main(ASMain.java:30)
Caused by: java.lang.Exception: Manifest error: Missing attribute name -  sun.security.action,sun.security.internal.spec,sun.security.jgss,
	at org.apache.felix.framework.cache.BundleCache.getMainAttributes(BundleCache.java:352)
	at org.apache.felix.framework.cache.BundleCache.getMainAttributes(BundleCache.java:199)
	at org.apache.felix.framework.cache.JarRevision.getManifestHeader(JarRevision.java:103)
	at org.apache.felix.framework.util.Util.getMultiReleaseAwareManifestHeaders(Util.java:993)
	at org.apache.felix.framework.BundleImpl.createRevision(BundleImpl.java:1266)
	at org.apache.felix.framework.BundleImpl.<init>(BundleImpl.java:113)
	at org.apache.felix.framework.Felix.installBundle(Felix.java:3260)
```

This exception should be investigated and fixed.

--- a/appserver/persistence/oracle-jdbc-driver-packages/pom.xml
+++ b/appserver/persistence/oracle-jdbc-driver-packages/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0, which is available at
@@ -19,17 +19,18 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    
     <parent>
         <groupId>org.glassfish.main.persistence</groupId>
         <artifactId>persistence</artifactId>
         <version>6.0.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
     </parent>
+    
     <artifactId>glassfish-oracle-jdbc-driver-packages</artifactId>
-    <packaging>jar</packaging>
+    
     <name>GlassFish Oracle JDBC Packages</name>
     <description>This bundle extends System Bundle to provide access to additional classes available in oracle jdbc driver.</description>
-    
+
     <developers>
         <developer>
             <id>mm110999</id>
@@ -40,38 +41,46 @@
             </roles>
         </developer>
     </developers>
-    
+
     <build>
         <plugins>
             <plugin>
-                <artifactId>maven-jar-plugin</artifactId>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                    </execution>
+                </executions>
                 <configuration>
-                    <archive>
-                        <manifestEntries>
-                            <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
-                            <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
-                            <Fragment-Host>system.bundle; extension:=framework</Fragment-Host>
-                            <Export-Package>
- oracle.jdbc,
- oracle.jdbc.internal,
- oracle.jdbc.pool,
- oracle.xdb,
- oracle.xdb.dom,
- oracle.sql,
- oracle.spatial.geometry,
- oracle.dms.instrument,
- oracle.dms.spy,
- oracle.xml.jaxp,
- oracle.xml.parser.schema,
- oracle.xml.parser.v2,
- oracle.ucp.jdbc,
- oracle.jdbc.dcn,
- oracle.AQ,
- oracle.kv
-                            </Export-Package>
-                            <Bundle-Description>${project.description}</Bundle-Description> 
-                        </manifestEntries>
-                    </archive>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Description>${project.description}</Bundle-Description>
+                        <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
+                        <Fragment-Host>system.bundle; extension:=framework</Fragment-Host>
+                        <Export-Package>
+                            oracle.jdbc,
+                            oracle.jdbc.internal,
+                            oracle.jdbc.pool,
+                            oracle.xdb,
+                            oracle.xdb.dom,
+                            oracle.sql,
+                            oracle.spatial.geometry,
+                            oracle.dms.instrument,
+                            oracle.dms.spy,
+                            oracle.xml.jaxp,
+                            oracle.xml.parser.schema,
+                            oracle.xml.parser.v2,
+                            oracle.ucp.jdbc,
+                            oracle.jdbc.dcn,
+                            oracle.AQ,
+                            oracle.kv
+                        </Export-Package>
+                    </instructions>
                 </configuration>
             </plugin>
         </plugins>

--- a/nucleus/core/extra-jre-packages/pom.xml
+++ b/nucleus/core/extra-jre-packages/pom.xml
@@ -1,23 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
+<!-- Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved. This program and the 
+    accompanying materials are made available under the terms of the Eclipse Public License v. 2.0, which 
+    is available at http://www.eclipse.org/legal/epl-2.0. This Source Code may also be made available under 
+    the following Secondary Licenses when the conditions for such availability set forth in the Eclipse Public 
+    License v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU Classpath Exception, 
+    which is available at https://www.gnu.org/software/classpath/license.html. SPDX-License-Identifier: EPL-2.0 
+    OR GPL-2.0 WITH Classpath-exception-2.0 -->
 
-    Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved.
-
-    This program and the accompanying materials are made available under the
-    terms of the Eclipse Public License v. 2.0, which is available at
-    http://www.eclipse.org/legal/epl-2.0.
-
-    This Source Code may also be made available under the following Secondary
-    Licenses when the conditions for such availability set forth in the
-    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
-    version 2 with the GNU Classpath Exception, which is available at
-    https://www.gnu.org/software/classpath/license.html.
-
-    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-
--->
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.glassfish.main.core</groupId>
@@ -29,7 +21,7 @@
     <packaging>jar</packaging>
     <name>GlassFish Extra JRE Packages</name>
     <description>This bundle extends System Bundle to provide access to additional classes available in JRE</description>
-    
+
     <developers>
         <developer>
             <id>ss141213</id>
@@ -40,96 +32,103 @@
             </roles>
         </developer>
     </developers>
-    
+
     <build>
         <plugins>
             <plugin>
-                <artifactId>maven-jar-plugin</artifactId>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                    </execution>
+                </executions>
                 <configuration>
-                    <archive>
-                        <manifestEntries>
-                            <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
                             <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
                             <Fragment-Host>system.bundle; extension:=framework</Fragment-Host>
-                            <Export-Package>
- com.ibm.security.util,
- com.ibm.security.x509,
- com.sun.corba.se.impl.orbutil.threadpool,
- com.sun.corba.se.spi.orbutil.threadpool,
- com.sun.j3d.loaders,
- com.sun.j3d.loaders.lw3d,
- com.sun.j3d.loaders.objectfile,
- com.sun.j3d.utils.behaviors.keyboard,
- com.sun.j3d.utils.behaviors.mouse,
- com.sun.j3d.utils.geometry,
- com.sun.j3d.utils.image,
- com.sun.j3d.utils.timer,
- com.sun.java.swing.plaf.windows,
- com.sun.jndi.cosnaming,
- com.sun.jndi.ldap,
- com.sun.mirror.apt,
- com.sun.mirror.declaration,
- com.sun.mirror.type,
- com.sun.mirror.util,
- com.sun.net.httpserver,
- com.sun.org.apache.xalan.internal.res,
- com.sun.org.apache.xalan.internal.xsltc.trax,
- com.sun.org.apache.xerces.internal.dom,
- com.sun.org.apache.xerces.internal.impl,
- com.sun.org.apache.xerces.internal.impl.dv.util,
- com.sun.org.apache.xerces.internal.jaxp,
- com.sun.org.apache.xerces.internal.parsers,
- com.sun.org.apache.xerces.internal.util,
- com.sun.org.apache.xerces.internal.xni,
- com.sun.org.apache.xerces.internal.xni.grammars,
- com.sun.org.apache.xerces.internal.xni.parser,
- com.sun.org.apache.xml.internal.dtm,
- com.sun.org.apache.xml.internal.serialize,
- com.sun.org.apache.xml.internal.utils,
- com.sun.org.apache.xpath.internal,
- com.sun.org.apache.xpath.internal.compiler,
- com.sun.org.apache.xpath.internal.functions,
- com.sun.org.apache.xpath.internal.jaxp,
- com.sun.org.apache.xpath.internal.objects,
- com.sun.security.auth,
- com.sun.security.auth.login,
- com.sun.security.auth.callback,
- com.sun.tools.apt,
- com.sun.tools.javac,
- org.omg.stub.java.rmi,
- org.w3c.dom.css,
- org.w3c.dom.html,
- org.w3c.dom.ranges,
- org.w3c.dom.traversal,
- org.w3c.dom.views,
- sun.awt.shell,
- sun.corba,
- sun.misc,
- sun.net.www,
- sun.nio.cs,
- sun.reflect,
- sun.rmi.rmic,
- sun.rmi.transport,
- sun.security.action,
- sun.security.internal.spec,
- sun.security.jgss,
- sun.security.jgss.spi,
- sun.security.krb5,
- sun.security.krb5.internal.crypto,
- com.sun.security.jgss,
- sun.security.provider,
- sun.security.ssl,
- sun.security.tools,
- sun.security.util,
- sun.security.x509,
- sun.security.pkcs,
- sun.tools.javac,
- sunw.io,
- sunw.util
-                            </Export-Package>
-                            <Bundle-Description>${project.description}</Bundle-Description> 
-                        </manifestEntries>
-                    </archive>
+            
+                            <Bundle-Description>${project.description}</Bundle-Description>
+                              <Export-Package>com.ibm.security.util,
+                                com.ibm.security.x509,
+                                com.sun.corba.se.impl.orbutil.threadpool,
+                                com.sun.corba.se.spi.orbutil.threadpool,
+                                com.sun.j3d.loaders,
+                                com.sun.j3d.loaders.lw3d,
+                                com.sun.j3d.loaders.objectfile,
+                                com.sun.j3d.utils.behaviors.keyboard,
+                                com.sun.j3d.utils.behaviors.mouse,
+                                com.sun.j3d.utils.geometry,
+                                com.sun.j3d.utils.image,
+                                com.sun.j3d.utils.timer,
+                                com.sun.java.swing.plaf.windows,
+                                com.sun.jndi.cosnaming,
+                                com.sun.jndi.ldap,
+                                com.sun.mirror.apt,
+                                com.sun.mirror.declaration,
+                                com.sun.mirror.type,
+                                com.sun.mirror.util,
+                                com.sun.net.httpserver,
+                                com.sun.org.apache.xalan.internal.res,
+                                com.sun.org.apache.xalan.internal.xsltc.trax,
+                                com.sun.org.apache.xerces.internal.dom,
+                                com.sun.org.apache.xerces.internal.impl,
+                                com.sun.org.apache.xerces.internal.impl.dv.util,
+                                com.sun.org.apache.xerces.internal.jaxp,
+                                com.sun.org.apache.xerces.internal.parsers,
+                                com.sun.org.apache.xerces.internal.util,
+                                com.sun.org.apache.xerces.internal.xni,
+                                com.sun.org.apache.xerces.internal.xni.grammars,
+                                com.sun.org.apache.xerces.internal.xni.parser,
+                                com.sun.org.apache.xml.internal.dtm,
+                                com.sun.org.apache.xml.internal.serialize,
+                                com.sun.org.apache.xml.internal.utils,
+                                com.sun.org.apache.xpath.internal,
+                                com.sun.org.apache.xpath.internal.compiler,
+                                com.sun.org.apache.xpath.internal.functions,
+                                com.sun.org.apache.xpath.internal.jaxp,
+                                com.sun.org.apache.xpath.internal.objects,
+                                com.sun.security.auth,
+                                com.sun.security.auth.login,
+                                com.sun.security.auth.callback,
+                                com.sun.tools.apt,
+                                com.sun.tools.javac,
+                                org.omg.stub.java.rmi,
+                                org.w3c.dom.css,
+                                org.w3c.dom.html,
+                                org.w3c.dom.ranges,
+                                org.w3c.dom.traversal,
+                                org.w3c.dom.views,
+                                sun.awt.shell,
+                                sun.corba,
+                                sun.misc,
+                                sun.net.www,
+                                sun.nio.cs,
+                                sun.reflect,
+                                sun.rmi.rmic,
+                                sun.rmi.transport,
+                                sun.security.action,
+                                sun.security.internal.spec,
+                                sun.security.jgss,
+                                sun.security.jgss.spi,
+                                sun.security.krb5,
+                                sun.security.krb5.internal.crypto,
+                                com.sun.security.jgss,
+                                sun.security.provider,
+                                sun.security.ssl,
+                                sun.security.tools,
+                                sun.security.util,
+                                sun.security.x509,
+                                sun.security.pkcs,
+                                sun.tools.javac,
+                                sunw.io,
+                                sunw.util</Export-Package>
+                    </instructions>
                 </configuration>
             </plugin>
         </plugins>

--- a/nucleus/core/extra-jre-packages/pom.xml
+++ b/nucleus/core/extra-jre-packages/pom.xml
@@ -1,24 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved. This program and the 
-    accompanying materials are made available under the terms of the Eclipse Public License v. 2.0, which 
-    is available at http://www.eclipse.org/legal/epl-2.0. This Source Code may also be made available under 
-    the following Secondary Licenses when the conditions for such availability set forth in the Eclipse Public 
-    License v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU Classpath Exception, 
-    which is available at https://www.gnu.org/software/classpath/license.html. SPDX-License-Identifier: EPL-2.0 
-    OR GPL-2.0 WITH Classpath-exception-2.0 -->
+<!--
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    
     <parent>
         <groupId>org.glassfish.main.core</groupId>
         <artifactId>nucleus-core</artifactId>
         <version>6.0.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
     </parent>
+    
     <artifactId>glassfish-extra-jre-packages</artifactId>
-    <packaging>jar</packaging>
+
     <name>GlassFish Extra JRE Packages</name>
     <description>This bundle extends System Bundle to provide access to additional classes available in JRE</description>
 

